translate/storage/tiki.py

```diff
@@ -66,7 +68,7 @@ def __init__(self, source=None, encoding="UTF-8"):
         self.location = []
         super(TikiUnit, self).__init__(source)
 
-    def __unicode__(self):
+    def __str__(self):
```

Comment:
I am a bit worried that this effectively overrides the `__str__()` method of `base.TranslationUnit`. I would be ok with this if we can tell for sure we don't need the base implementation anywhere, but I am afraid that is a long shot.